fog/sample-paykit/proto/remote_wallet.proto

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2018-2021 The MobileCoin Foundation
+
+/// This file contains a simple service wrapper for a wallet that is used by the fog conformance tests
+/// to perform balance checks against an arbitrary client implementation.
+/// The sample-paykit includes such implementation, and the idea is that other clients/SDKs could
+/// also implement this service and then get tested by the fog conformance tests Python script.
+
+syntax = "proto3";
+
+package remote_wallet;
+
+option java_package = "com.example.myapplication.proto";
+option java_outer_classname = "RemoteWalletProto";
+
+import "google/protobuf/empty.proto";
+
+service RemoteWalletApi {
+    /// Start a new client and perform a fresh balance check.
+    rpc FreshBalanceCheck (FreshBalanceCheckRequest) returns (BalanceCheckResponse);
+
+    /// Perform a balance check on a previously-started client.
+    rpc FollowupBalanceCheck (FollowupBalanceCheckRequest) returns (BalanceCheckResponse);
+
+    /// Stop a previously-started client.
+    rpc Stop (StopRequest) returns (google.protobuf.Empty);
+
+    /// Get debug info for a previously-started client.
+    rpc Debug (DebugRequest) returns (DebugResponse);
+
+    /// Drop all previously-started clients.
+    rpc Reset (google.protobuf.Empty) returns (google.protobuf.Empty);
+}
+
+message FreshBalanceCheckRequest {
+    /// Root entropy for the account the new client will be using.
+    bytes root_entropy = 1;
+
+    /// View server URI
+    string view_server_uri = 2;
+
+    /// Ledger server URI
+    string ledger_server_uri = 3;
```

Comment:
This should be unified into a single `fog_uri`.

Reply:
That complicates the local setup. As a reminder, this script runs against services running locally and not a deployed network. If we wanted it to be a single URI then we would need to run `nginx` or something like that to do the splitting, and I am hoping we can avoid that. 
Are the SDKs not going to support specifying separate URLs? @voloshyn did provide me with a version that supports that on Android.

Reply:
The Swift SDK has already dropped support for separating the Fog services.

Aren't there 4 ledger services though, with 2 of them being attested? Could fog view be added to that setup in some way so that it's 5 services?

And ideally fog report would be on the same url as well, although currently it doesn't look like the conformance tests test that part of the system. (But I imagine we will want to expand the conformance tests over time.)

Reply:
There are currently two server binaries (view and ledger), each listening on their own port. Its true that report is also a separate server but at the moment it is not used in the tests. Similarly to what I said in the other comment, this is currently how `master` works. I would prefer to keep the changes in this PR minimal and move to the single URL+nginx in a separate PR.